client/lib/signup/step-actions.js

```diff
@@ -358,24 +359,23 @@ function processItemCart(
 	themeSlugWithRepo
 ) {
 	const addToCartAndProceed = () => {
-		let privacyItem = null;
 		const state = reduxStore.getState();
-		const { domainItem } = newCartItems;
```

Comment:
All calls to this function, `processItemCart`, pass `newCartItems` as an array, so I'm not sure why object de-structuring would be used here

- https://github.com/Automattic/wp-calypso/blob/1e13f50657b629e53e2d708e053e158214e22c4b/client/lib/signup/step-actions.js#L208
- https://github.com/Automattic/wp-calypso/blob/1e13f50657b629e53e2d708e053e158214e22c4b/client/lib/signup/step-actions.js#L337
- https://github.com/Automattic/wp-calypso/blob/1e13f50657b629e53e2d708e053e158214e22c4b/client/lib/signup/step-actions.js#L347